pysite/logs.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+
+
+class Logs:
```

Comment:
Do we really need this class? I'd have just used the `logging` module directly, I don't see anything we get from this class that the module doesn't already provide

Reply:
this is a very astute point. see the comments on the PR itself as well as #dev-code for more discussion on this topic.

Reply:
Unfortunately I have to override the flask logger due to the nature of gunicorn and flask, which is why I instantiate a new logger object from werkzeug. This lets the logging pass through to gunicorn and out to stdout.

Reply:
Does werkzeug not use the standard python logging stuff? If it does, all you should need is the name of the logger, then you can get a handle to it